cfme/tests/containers/test_static_custom_attributes.py

```diff
@@ -131,14 +131,12 @@ def test_add_attribute_with_empty_name(provider):
         * You should get an error
         * You should not see this attribute in the custom  attributes table
     """
-    try:
+    with pytest.raises(APIException):
         provider.add_custom_attributes(
             CustomAttribute('', "17")
         )
         pytest.fail('You have added custom attribute with empty name'
```

Comment:
The current message doesn't provide much additional context, if you want some more context or assertion behavior on failure check out the examples again, they have 1-2 with an assert following a pytest.raises  - but its not inside the `with():` block.